tracer/src/Datadog.Trace.Tools.dd_dotnet/Checks/IRegistryService.cs

```diff
@@ -0,0 +1,16 @@
+// <copyright file="IRegistryService.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+namespace Datadog.Trace.Tools.dd_dotnet.Checks
+{
+    internal interface IRegistryService
+    {
+        string[] GetLocalMachineValueNames(string key);
+
+        string? GetLocalMachineValue(string key);
+    }
```

Comment:
Some of this stuff _is_ shared between the tools. I wonder if we should at least link across what we can, otherwise it's very likely we'll end up with the tools getting out of sync IMO. It's fine if we going to bundle dd-dotnet in dd-trace sooner rather than later, but I worry about Maximo's work being missed (for example)